app/src/forms/email/reminderService.js

```diff
@@ -0,0 +1,296 @@
+
+const { getAvailableDates } = require('../common/utils');
+const emailService  = require('./emailService');
+const moment = require('moment');
+const {  EmailTypes, ScheduleType } = require('../common/constants');
+const config = require('config');
+const log = require('../../components/log')(module.filename);
+const {  SubmissionData, UserFormAccess, Form } = require('../common/models');
+const { Roles } = require('../common/constants');
+
+const service = {
+  _init: async ()=> {
+    const forms = await service._getForms();
+
+    const q     = await service._getReminders(forms);
+
+    const referer = service._getReferer();
+    const resolve = [];
+    const errors = [];
+    let mail = 0;
+    
+    if(q.length!==0) {
+      for(let i = 0 ; i < q.length ; i++) {
+        if(!q[i].error){
+          const obj = await  service._runQueries(q[i].statement);
+          let result = await service._initStatement(obj);
+          let chesResponses = service._initMaillSender(result, referer);
+          resolve.push({
+            formId:result.form.id,
+            formName :  result.form.name,
+            type_mail : result.state,
+            number_mail: result.submitters.length,
+            period_type : q[i].periodType,
+            chesResponses
+          });
+          // eslint-disable-next-line no-unused-vars
+          mail+= result.submitters.length;
+        } else{
+          errors.push(q[i].message);
+        }
+      }
+      return {
+        messsage: `${q.length} Forms found , ${mail} mails sent.`,
+        errors,
+        resolve
+      };
+    }
+    else {
+      return {
+        messsage: '0 email sent',
```

Comment:
Message is spelled incorrectly, it should be `message` and not `messsage`.

The message itself should also say `0 emails sent.`

Reply:
This is an English grammar reason, see https://www.bbc.co.uk/worldservice/learningenglish/grammar/learnit/learnitv354.shtml